res/eg/convert_to_tutorial.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# usage:
+#  ./res/eg/convert_to_tutorial.sh res/eg/1d/jupyter/weak/weak.ipynb
+#
+
+set -ex
+
+jupyter nbconvert --to notebook $1  \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)